contracts/src/starknet/rewaave/token_bridge.cairo

```diff
@@ -219,5 +227,20 @@ func handle_deposit{syscall_ptr : felt*, pedersen_ptr : HashBuiltin*, range_chec
     IERC20.mint(contract_address=l2_token, recipient=l2_recipient, amount=amount)
     deposit_handled.emit(l2_token, l2_recipient, amount)
 
+    deposit_handled.emit(l2_token, l2_recipient, amount)
```

Comment:
Two emits?

Reply:
My bad, it has probably been added through a git rebase

Reply:
My bad, it has probably been added through a git rebase